core/lib/instrument.c

```diff
@@ -1811,6 +1811,7 @@ create_and_initialize_module_data(app_pc start, app_pc end, app_pc entry_point,
             copy->segments[i].start = os_segments[i].start;
```

Comment:
Also, please add a feature entry to https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L230

Reply:
Done.